subgraphs/abracadabra/src/common/getters.ts

```diff
@@ -205,7 +205,7 @@ export function getOrCreateLendingProtocol(): LendingProtocol {
   LendingProtocolEntity.subgraphVersion = "0.0.6";
```

Comment:
subgraph version should be at least 1.0.0 since it's going to be deployed now

Reply:
fixed